src/NATS.Client.Core/NatsConnection.cs

```diff
@@ -92,6 +92,8 @@ public NatsConnection(NatsOptions options)
 
     internal string InboxPrefix { get; }
 
+    public string NewInbox() => $"{InboxPrefix}{Guid.NewGuid():n}";
```

Comment:
This used to be an extension method too. I'm not sure if it's necessary to keep this public. If so, it should be part of the interface.

Reply:
You're right. It should be internal or not even be exposed here. Please feel free to leave this one out. I don't think It'd be necessary for unit testing.

Reply:
Done in 37a4b00fcf772e0666eb78d6e387b3c5160c3ffa